### PR TITLE
Capitalize FORD tag comments in derivs/

### PR DESCRIPTION
**Pull request by vanderhe**
Ran [capitalize_ford_comments](https://github.com/dftbplus/dftbplus/blob/main/utils/srcmanip/capitalize_ford_comments) in src/dftbp/derivs/.

### CODE REVIEW

File: src/dftbp/derivs/linearresponse.F90 [hunk 1]
```diff
@@ -80,10 +80,10 @@ subroutine dRhoReal(env, dHam, neighbourList, nNeighbourSK, iSparseStart, img2Ce
     !> Particular spin/k-point
     integer, intent(in) :: iKS
 
-    !> K-points and spins to process
+    !> k-points and spins to process
```
Comment (bhourahine) [inline src/dftbp/derivs/linearresponse.F90:83]: Interesting question, as this is usually a lower case mathematical variable, and does look a bit weird at the start of a line.
```suggestion
    !> The k-points and spins to process
```

File: src/dftbp/derivs/linearresponse.F90 [hunk 2]
```diff
@@ -493,19 +493,19 @@ subroutine dRhoFermiChangeReal(dRhoExtra, env, maxFill, parallelKS, iKS, neighbo
     !> Maximum allowed number of electrons in a single particle state
     real(dp), intent(in) :: maxFill
 
-    !> K-points and spins to process
+    !> k-points and spins to process
```
Comment (bhourahine) [inline src/dftbp/derivs/linearresponse.F90:496]:
```suggestion
    !> The k-points and spins to process
```

File: src/dftbp/derivs/linearresponse.F90 [hunk 3]
```diff
@@ -658,16 +658,16 @@ subroutine dRhoCmplx(env, dHam, neighbourList, nNeighbourSK, iSparseStart, img2C
...
 
-    !> map from image atoms to the original unique atom
+    !> Map from image atoms to the original unique atom
     integer, intent(in) :: img2CentCell(:)
 
     !> Dense matrix descriptor
     type(TDenseDescr), intent(in) :: denseDesc
 
-    !> K-points and spins to process
+    !> k-points and spins to process
```
Comment (bhourahine) [inline src/dftbp/derivs/linearresponse.F90:667]:
```suggestion
    !> The k-points and spins to process
```

File: src/dftbp/derivs/linearresponse.F90 [hunk 4]
```diff
@@ -688,7 +688,7 @@ subroutine dRhoCmplx(env, dHam, neighbourList, nNeighbourSK, iSparseStart, img2C
     !> Atomic orbital information
     type(TOrbitals), intent(in) :: orb
 
-    !> returning dRhoSparse on exit
+    !> Returning dRhoSparse on exit
     real(dp), intent(inout) :: dRhoSparse(:,:)
 
     !> k-points
```
Comment (bhourahine) [inline src/dftbp/derivs/linearresponse.F90:694]:
```suggestion
    !> The k-points
```

File: src/dftbp/derivs/linearresponse.F90 [hunk 5]
```diff
@@ -1049,10 +1049,10 @@ subroutine dRhoFermiChangeCmplx(dRhoExtra, env, maxFill, parallelKS, iKS, kPoint
     !> Maximum allowed number of electrons in a single particle state
     real(dp), intent(in) :: maxFill
 
-    !> K-points and spins to process
+    !> k-points and spins to process
```
Comment (bhourahine) [inline src/dftbp/derivs/linearresponse.F90:1052]:
```suggestion
    !> The k-points and spins to process
```

File: src/dftbp/derivs/linearresponse.F90 [hunk 6]
```diff
@@ -1049,10 +1049,10 @@ subroutine dRhoFermiChangeCmplx(dRhoExtra, env, maxFill, parallelKS, iKS, kPoint
...
     type(TParallelKS), intent(in) :: parallelKS
 
-    !> spin/kpoint channel
+    !> Spin/kpoint channel
     integer, intent(in) :: iKS
 
     !> k-points
```
Comment (bhourahine) [inline src/dftbp/derivs/linearresponse.F90:1058]:
```suggestion
    !> The k-points
```

File: src/dftbp/derivs/linearresponse.F90 [hunk 7]
```diff
@@ -1229,16 +1229,16 @@ subroutine dRhoPauli(env, dHam, idHam, neighbourList, nNeighbourSK, iSparseStart
...
 
-    !> map from image atoms to the original unique atom
+    !> Map from image atoms to the original unique atom
     integer, intent(in) :: img2CentCell(:)
 
     !> Dense matrix descriptor
     type(TDenseDescr), intent(in) :: denseDesc
 
-    !> K-points and spins to process
+    !> k-points and spins to process
```
Comment (bhourahine) [inline src/dftbp/derivs/linearresponse.F90:1238]:
```suggestion
    !> The k-points and spins to process
```

File: src/dftbp/derivs/linearresponse.F90 [hunk 8]
```diff
@@ -1259,10 +1259,10 @@ subroutine dRhoPauli(env, dHam, idHam, neighbourList, nNeighbourSK, iSparseStart
...
 
-    !> returning dRhoSparse on exit
+    !> Returning dRhoSparse on exit
     real(dp), intent(inout) :: dRhoSparse(:,:)
 
-    !> returning imaginary part of dRhoSparse on exit
+    !> Returning imaginary part of dRhoSparse on exit
     real(dp), intent(inout), allocatable :: idRhoSparse(:,:)
 
     !> k-points
```
Comment (bhourahine) [inline src/dftbp/derivs/linearresponse.F90:1268]:
```suggestion
    !> The k-points
```

File: src/dftbp/derivs/linearresponse.F90 [hunk 9]
```diff
@@ -1619,10 +1619,10 @@ subroutine dRhoFermiChangePauli(dRhoExtra, idRhoExtra, env, parallelKS, iKS, kPo
     !> Environment settings
     type(TEnvironment), intent(inout) :: env
 
-    !> K-points and spins to process
+    !> k-points and spins to process
```
Comment (bhourahine) [inline src/dftbp/derivs/linearresponse.F90:1622]:
```suggestion
    !> The k-points and spins to process
```

File: src/dftbp/derivs/linearresponse.F90 [hunk 10]
```diff
@@ -1619,10 +1619,10 @@ subroutine dRhoFermiChangePauli(dRhoExtra, idRhoExtra, env, parallelKS, iKS, kPo
...
     type(TParallelKS), intent(in) :: parallelKS
 
-    !> spin/kpoint channel
+    !> Spin/kpoint channel
     integer, intent(in) :: iKS
 
     !> k-points
```
Comment (bhourahine) [inline src/dftbp/derivs/linearresponse.F90:1628]:
```suggestion
    !> The k-points
```

File: src/dftbp/derivs/perturb.F90 [hunk 11]
```diff
@@ -186,7 +186,7 @@ subroutine wrtEField(this, env, parallelKS, filling, eigvals, eigVecsReal, eigVe
     !> Environment settings
     type(TEnvironment), intent(inout) :: env
 
-    !> K-points and spins to process
+    !> k-points and spins to process
```
Comment (bhourahine) [inline src/dftbp/derivs/perturb.F90:189]:
```suggestion
    !> The k-points and spins to process
```

File: src/dftbp/derivs/perturb.F90 [hunk 12]
```diff
@@ -500,10 +500,10 @@ subroutine wrtVAtom(this, env, parallelKS, isAutotestWritten, autotestTagFile,&
...
```
Comment (bhourahine) [inline src/dftbp/derivs/perturb.F90:503]:
```suggestion
    !> The k-points and spins to process
```

File: src/dftbp/derivs/perturb.F90 [hunk 13]
```diff
@@ -879,7 +879,7 @@ subroutine response(env, parallelKS, dPotential, nAtom, orb, species, neighbourL
...
```
Comment (bhourahine) [inline src/dftbp/derivs/perturb.F90:882]:
```suggestion
    !> The k-points and spins to process
```

File: src/dftbp/derivs/perturb.F90 [hunk 14]
```diff
@@ -1528,7 +1528,7 @@ subroutine init_perturbation(parallelKS, tolDegen, nOrbs, nKpts, nSpin, nIndepHa
       & dRhoInSqr, dRhoOutSqr, dPotential, orb, nAtom, tMetallic, neFermi, eigvals, tempElec, Ef,&
       & kWeight)
 
-    !> K-points and spins to process
+    !> k-points and spins to process
```
Comment (bhourahine) [inline src/dftbp/derivs/perturb.F90:1531]:
```suggestion
    !> The k-points and spins to process
```

**Review by bhourahine — APPROVED** [review]
All OK, but see suggestion about k-points, also Nr. -> Number might be worth doing.

File: src/dftbp/derivs/linearresponse.F90 (hunk 1 above)
Comment (vanderhe) [reply src/dftbp/derivs/linearresponse.F90:83]: True, thanks for the suggestion, I will just apply it. Of course this particular case occurs all over the codebase, so let's remember this convention and successively apply it.

**Pull request merged**